data/_ui/tooltips.txt

```diff
@@ -1458,6 +1458,9 @@ tip "Interrupt fast-forward"
 tip "Landing zoom"
 	`Apply a cinematic zoom in and out when you are landing and taking off.`
 
+tip "Half speed"
+	`Reduces FPS to 30, halving the speed of the game.`
```

Comment:
We probably also should tell players that damage per second and other similar things are still calculated for normal game speed (aren't they?).

Reply:
They are, time literally flows half as fast. Do you have a suggestion for a good way to put that?

Reply:
That sounds actively confusing. I don't think it really needs an explanation. Half speed means half speed.